src/Validator/Rules/CustomValidationRule.php

```diff
@@ -7,7 +7,10 @@
 use GraphQL\Validator\ValidationContext;
 
 /**
- * @phpstan-type VisitorFn callable(ValidationContext): (array<string, callable(Node): VisitorOperation|mixed|null>|array<string, array<string, callable(Node): VisitorOperation|mixed|null>>)
```

Comment:
There is like no support for using parentheses for defining precedence in union/intersection definitions

https://github.com/phpstan/phpdoc-parser/issues/74